src/styles/block-style/page-style.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  margin: 0;
```

Comment:
Don't reset margin or padding in * selector, reset only for elements you are using